src/app.js

```diff
@@ -6,8 +6,282 @@ import _ from 'underscore';
+    const inputElement = $(`#reserve-trip input[name="${ field }"]`);
+    const value = inputElement.val();
+
+    reservationData[field] = value;
+  });
+
+  console.log("Read reservation data");
+  console.log(reservationData);
+
+  return reservationData;
+};
+
+const readAddFormData = function readAddFormData() {
+  const tripData = {};
+
+  TRIP_FIELDS.forEach((field) => {
+    const inputElement = $(`#add-trip-form input[name="${ field }"]`);
+    const value = inputElement.val();
+
+    tripData[field] = value;
+  });
+
+  console.log("Read trip data");
+  console.log(tripData);
+
+  return tripData;
+};
+
+//////////////////////status messages///////////////////////
+const handleStatusMessages = (messages) => {
+  console.log('In handleStatusMessages()');
+  console.log(messages);
+
+  $('#status-messages div').empty();
+
+  let generatedHtml = statusMessagesTemplate(messages);
+  console.log(generatedHtml);
+  let messagesElem = $('#status-messages div');
+  console.log(messagesElem);
+  messagesElem.append(generatedHtml);
+  $('#status-messages').show();
+};
+
+const clearStatus = function clearStatus() {
+  $('#status-messages p').html('');
+  $('#status-messages').hide();
+};
+
+const clearAddTripForm = function clearAddTripForm() {
+  $('#trip-form-message').hide();
+  // $('#trip-form-button').hide();
+  $('#add-trip-form-close').hide();
+}
+
+const clearAllTripsTable = function clearAllTripsTable() {
+  $('#trip-table').hide();
+  $('#trip-table-close').hide();
+  $('#filter-triplist').hide();
+}
+
+const clearIndividualTripDetails = function clearIndividualTripDetails() {
+  $('#individual-trip-details').hide();
+  $('#individual-trip-details-close').hide();
+}
+////////////////eventHandlers////////////////////
+const showIndividualTripHandler = function showIndividualTripHandler(event) {
+  console.log('in the trip click');
+  $('html,body').scrollTop(0);
+
+  let tripId = $(this).attr('data-id');
+  let trip = tripList.get(tripId)
+    trip.fetch({
+      success: function(model) {
+        const individualtripListElement = $('#individual-trip-details');
+
+        individualtripListElement.html('');
+
+        const generatedHTMLTripDetails = individualTripTemplate(model.attributes);
+        individualtripListElement.append(generatedHTMLTripDetails);
+
+        $('#individual-trip-details').show();
+        $('#individual-trip-details-close').show();
+
+        // Listen for submit event on #reserve-trip
+        $('#reserve-trip').on('submit', addReservationHandler);
+      }
+    });
+  };
+
+const addTripHandler = function(event) {
+  event.preventDefault();
+  const trip = new Trip(readAddFormData());
+
+  if (!trip.isValid()) {
+    console.log('Client side error handling');
+    handleStatusMessages(trip.validationError);
+    return;
+  }
+
+  trip.save({}, {
+    success: (model, response) => {
+      console.log('Successfully saved trip!');
+      tripList.add(trip);
+      $('#trip-form-message').hide();
```

Comment:
I like how you're hiding the form after a successful save.

How about clearing the form entries when you do so as well like:
```
$('input[name=about]').val('');
```